### ISSUE TITLE
How do I build this project?

### ISSUE THREAD

**Issue by kevburnsjr** [ydnar/wasm-tools-go#84]
There is no `Makefile` and there are no instructions for building the binary.

I tried `go build` and `go install` but neither worked.

```
> go build cmd/wit-bindgen-go/*.go

cmd/wit-bindgen-go/main.go:8:2: no required module provides package github.com/urfave/cli/v3; to add it:
        go get github.com/urfave/cli/v3
cmd/wit-bindgen-go/main.go:10:2: no required module provides package github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/generate; to add it:
        go get github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/generate
cmd/wit-bindgen-go/main.go:11:2: no required module provides package github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/print; to add it:
        go get github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/print
cmd/wit-bindgen-go/main.go:12:2: no required module provides package github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/wit; to add it:
        go get github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go/cmd/wit
```
```
> go install github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go@latest

go: downloading github.com/ydnar/wasm-tools-go v0.0.0-20240419173811-42579b015632
go: github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go@latest (in github.com/ydnar/wasm-tools-go@v0.0.0-20240419173811-42579b015632):
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

How to build? 🤷‍♂️

**Comment by rajatjindal** [comment]
I tried, and this should work `go build -o wit-bindgen-go cmd/wit-bindgen-go/main.go`

**Comment by ydnar** [comment]
Working on a fix.

**Comment by ydnar** [comment]
This should be fixed in #85.

**Comment by kevburnsjr** [comment]
Success! Thank you.
```
> go install github.com/ydnar/wasm-tools-go/cmd/wit-bindgen-go@latest
go: downloading github.com/ydnar/wasm-tools-go v0.0.0-20240422194223-39a9b509c4bd
```
